.github/workflows/semantic_release.yml

```diff
@@ -0,0 +1,70 @@
+# This workflow runs semantic release, bumps, generates changelog, and tags the project
+name: Semantic Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      dry-run:
+        description: "Are you releasing a new version? (Y/n)"
```

Comment:
I think the convention for CLIs is that a capital letter suggests it's a default argument. Should `Y` be lowercase `y` here?

Reply:
yup!